paf-mvp-core-js/src/express/node.ts

```diff
@@ -139,9 +139,9 @@ export class Node implements INode {
    * Begin handling of a request.
    * Should be called first.
    */
-  beginHandling = (req: Request, res: Response, next: NextFunction) => {
+  beginHandling = (req: any, res: Response, next: NextFunction) => {
```

Comment:
Any reason for this regression in type check? I guess it's a mistake after a refactor ;-)

Reply:
Not a mistake. `Request` type doesn't have a correlationId() method. This field is added by the express-correlation-id middleware.

Reply:
Then simply do this:
```typescript
beginHandling = (req: Request & {correlationId(): string}, res: Response, next: NextFunction) => {
```